### ISSUE TITLE
Max = min causes an unhandled exception

### ISSUE THREAD

**Issue by volgin** [matthewfx/sleek_circular_slider#5]
If max value equals min value, an unhandled exception is thrown by this code because it attempts to divide by zero:

```
  final divider = (max - min) / 100;
  return (value - previous).abs() ~/ divider * 15;
```

**Comment by matthewfx** [comment]
Thanks Volgin. I will be releasing an update soon and will address this bug in it too.

**Comment by matthewfx** [comment]
Fixed in 1.1.0
